src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -473,26 +496,73 @@ object EntityClient {
     override def metadataForEntity(entity: Entity): F[EntityMetadata] =
       for {
         token <- getAuthenticationToken
-
+        queryParams = Map("max" -> 1000, "start" -> 0)
         path <- me.fromOption(
           entity.path,
           PreservicaClientException(missingPathExceptionMessage(entity.ref))
         )
+        entityUrl = s"$apiUrl/$path/${entity.ref}"
+        entityType = entity.entityType.get
```

Comment:
I know it's unlikely that this will be empty but it'd be good to avoid the `.get` if possible.
There's a bit in the `entitiesByIdentifier` method that does 
```scala
me.fromOption(
      entity.entityType,
      PreservicaClientException(s"No entity type found for entity ${entity.ref}")
    )
```
It might be worth putting that in a method like `getEntityType` and then you can call that in both places. What do you think?